Assets/HoloToolkit/UX/Scripts/Buttons/CompoundButtonAnim.cs

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System;
+using System.Collections;
+
+namespace HoloToolkit.Unity.Buttons
+{
+    /// <summary>
+    /// Anim controller button offers as simple way to link button states to animation controller parameters
+    /// </summary>
+    [RequireComponent(typeof(CompoundButton))]
+    public class CompoundButtonAnim : MonoBehaviour
```

Comment:
Is this a duplicated class?

There's another class that looks oddly similar: `AnimControllerButton.cs`

Reply:
Lars wrote all the compound button stuff and yeah there's a decent amount of duplication though the CompondButtonAnim is not a button but meant to be an added set of functionality requiring a compound button.

AnimControllerButton
> Whole button that uses an animation controller to change state
CompoundButtonAnim
> Adds animation controller functionality to a CompoundButton

Reply:
I can't really discern the difference